index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Blog Page</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+
+  <nav class="nav__bar">
+    <h3 class="auth__name">Dustin Walker</h3>
+    <div class="nav-right">
+      <a class="links" href="#">Portfolio</a>
+      <a class="links" href="#">Blog</a>
+      <a class="links" href="#">Contact</a>
+    </div>
+
+  </nav>
+
+  <div class="container">
+    <div class="card_left">
```

Comment:
All of the "cards" should be able to use the same class and be within one single "container" element